AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -658,4 +664,117 @@ - (void)authTokenContext:(MSAuthTokenContext *)__unused authTokenContext didUpda
   }
 }
 
+#pragma mark - Reachability
+
+- (void)networkStateChanged:(NSNotificationCenter *)__unused notification {
+  //(void)notification;
```

Comment:
Remove commented line.  Also what will the notification be used for in the future?